Missing check for yaml-cpp library

Cmake configures successfully but during the build I get:

    [ 51%] Building CXX object cartographer_ros/CMakeFiles/map_writer.dir/map_writer.cc.o
    /home/zft/cartographer/cartographer_ros/cartographer_ros/cartographer_ros/map_writer.cc:22:27: fatal error: yaml-cpp/yaml.h: No such file or directory
     #include "yaml-cpp/yaml.h"
                               ^
    compilation terminated.
    cartographer_ros/CMakeFiles/map_writer.dir/build.make:62: recipe for target 'cartographer_ros/CMakeFiles/map_writer.dir/map_writer.cc.o' failed
    make[2]: *** [cartographer_ros/CMakeFiles/map_writer.dir/map_writer.cc.o] Error 1
    CMakeFiles/Makefile2:1615: recipe for target 'cartographer_ros/CMakeFiles/map_writer.dir/all' failed
    make[1]: *** [cartographer_ros/CMakeFiles/map_writer.dir/all] Error 2
    Makefile:138: recipe for target 'all' failed
    make: *** [all] Error 2

The CMakeLists.txt should not forget to check for the yaml-cpp library.

--- a/cartographer_ros/CMakeLists.txt
+++ b/cartographer_ros/CMakeLists.txt
@@ -28,7 +28,6 @@ set(PACKAGE_DEPENDENCIES
   tf2_ros
 )
 
-include(FindPkgConfig)
 
 find_package(cartographer REQUIRED)
 include("${CARTOGRAPHER_CMAKE_DIR}/functions.cmake")
@@ -36,6 +35,8 @@ google_initialize_cartographer_project()
 google_enable_testing()
 
 find_package(catkin REQUIRED COMPONENTS ${PACKAGE_DEPENDENCIES})
+
+include(FindPkgConfig)
 PKG_SEARCH_MODULE(YAMLCPP REQUIRED yaml-cpp>=0.5.1)
 
 catkin_package(

--- a/cartographer_ros/CMakeLists.txt
+++ b/cartographer_ros/CMakeLists.txt
@@ -28,12 +28,15 @@ set(PACKAGE_DEPENDENCIES
   tf2_ros
 )
 
+include(FindPkgConfig)
+
 find_package(cartographer REQUIRED)
 include("${CARTOGRAPHER_CMAKE_DIR}/functions.cmake")
 google_initialize_cartographer_project()
 google_enable_testing()
 
 find_package(catkin REQUIRED COMPONENTS ${PACKAGE_DEPENDENCIES})
+PKG_SEARCH_MODULE(YAMLCPP REQUIRED yaml-cpp>=0.5.1)
 
 catkin_package(
   CATKIN_DEPENDS

